CMSgov/cms-carts-seds
# MDCT-208: Certify and Submit

# Description

[MDCT-208: Certify and Submit](https://qmacbis.atlassian.net/browse/MDCT-208)

Allows a state user to certify and submit a report.

## How to test

* Run locally
* As a state user, certify a report.
* Ensure after submitting a report that the time and user and a "Return Home" button displays.
* Ensure the report becomes uneditable.
* Ensure the status on the homepage is "Certified and Submitted" and the Action is "View".
* Ensure that as an admin user you are not able to certify a report.

## Dependencies

None.

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
